_posts/patterns/2015-12-21-lists.md

```diff
@@ -0,0 +1,339 @@
+        <p class="item">Darkwell has a troubled economy, which is mainly supported by jewelcrafting, fishing and fletching. But their biggest strengths are intricate jewelcrafting and rare herbalism. However, Darkwell lacks people skilled in farming.</p>
+        <p class="item">Raised inside a geyser field, the port of Darkwell is home to vikings lead by Mrs. Chilson. This port wasn't built by a geyser field by accident, as it has rare resources, which is of great importance to the people of Darkwell and its success.</p>
+    </div>
+
+    <ol class="list-ordered no-indent">
+        <li class="item">Figs (No Indent)</li>
+        <li class="item">Oranges (No Indent)</li>
+        <li class="item">Plums (No Indent)</li>
+    </ol>
+</div>
+
+<h3 class="hd-6 example-set-hd">Definition Lists</h3>
+<div class="example-set">
+    <dl>
+        <dt class="hd-3">RSS (hd-3)</dt>
+        <dd>An XML format for aggregating information from websites whose 
+        content is frequently updated.</dd>
+        <dt class="hd-3">Weblog  (hd-3)</dt>
+        <dd>Contraction of the term "web log", a weblog is a 
+        website that is periodically updated, like a journal</dd>
+    </dl>
+
+    <dl>
+        <dt class="hd-6">Founders: (hd-6)</dt>
+        <dd class="copy copy-meta">Clark Kent (meta copy)</dd>
+        <dd class="copy copy-meta">Bruce Wayne (meta copy)</dd>
+        <dd class="copy copy-meta">Diana Prince (meta copy)</dd>
+        <dt class="hd-6">Members: (hd-6)</dt>
+        <dd class="copy copy-meta">Hal Jordan (meta copy)</dd>
+        <dd class="copy copy-meta">Barry Allen (meta copy)</dd>
+        <dd class="copy copy-meta">J'onn J'onzz (meta copy)</dd>
+    </dl>
+</div>
+
+<h3 class="hd-6 example-set-hd">Link-based Lists</h3>
+<div class="example-set">
+    <nav class="list-links">
+        <span class="item"><a href="http://ux.edx.org">Home</a></span>
+        <span class="item"><a href="http://ux.edx.org">About</a></span>
+        <span class="item"><a href="http://ux.edx.org">Jobs</a></span>
+        <span class="item"><a href="http://ux.edx.org">Team</a></span>
+    </nav>
+
+    <nav class="list-links list-inline">
+        <span class="item"><a href="http://ux.edx.org">Home</a></span>
+        <span class="item"><a href="http://ux.edx.org">About</a></span>
+        <span class="item"><a href="http://ux.edx.org">Jobs</a></span>
+        <span class="item"><a href="http://ux.edx.org">Team</a></span>
+    </nav>
+
+    <ul class="list-links">
+        <li class="item has-block-link">
+            <a href="http://ux.edx.org/">Apples (Block-level links)</a>
+        </li>
+        <li class="item has-block-link">
+            <a href="http://ux.edx.org/">Apricots (Block-level links)</a>
+        </li>
+        <li class="item has-block-link">
+            <a href="http://ux.edx.org/">Bananas (Block-level links)</a>
+        </li>
+    </ul>
+</div>
+
+<h3 class="hd-6 example-set-hd">List with Dividers</h3>
```

Comment:
I prefer these awesome stylings as the default for "unstyled" lists because they don't include the bullet (which some authors may not want) but do still visually indicate a list.